src/custom/pages/Claim/ClaimsTable.tsx

```diff
@@ -69,7 +83,7 @@ const ClaimsTableRow = ({
           Vesting: <b>{type === ClaimType.Airdrop ? 'No' : '4 years (linear)'}</b>
         </span>
         <span>
-          Ends in: <b>28 days, 10h, 50m</b>
+          Ends in: <b>{start && end && <Countdown start={start} end={end} />}</b>
```

Comment:
Not sure we want to do something special when either of those values are null (still loading) or the component returns null (means timeout is over)

Reply:
useSWR would work here or nah?

Reply:
Nah, not needed. The countdown is self contained in the component